gcstorage/gcstorage.go

```diff
@@ -0,0 +1,253 @@
+package gcstorage
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/goware/cachestore"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+var ErrNotSupported = fmt.Errorf("not supported")
```

Comment:
It might make sense to move this (sentinel) error to the top-level pkg

`return fmt.Errorf("gcstorage .SetEx(): %w", cachestore.ErrNotSupported)`